NEWS.md

```diff
@@ -28,7 +28,11 @@
   data tables).
 
 * We have begun the process of add translations for tidyr verbs beginning
-  with `pivot_wider()` (@markfairbanks, #189).
+  with:
+  
+  * `drop_na()` (@markfairbanks, #194)
```

Comment:
dtplyr just made it to CRAN so once you merge/rebase, you'll need to move this to a new bullet in the dev version.